cmd/launcher/launcher.go

```diff
@@ -379,7 +379,7 @@ func runLauncher(ctx context.Context, cancel func(), opts *launcher.Options) err
 	if k.Autoupdate() {
 		// Create a new TUF autoupdater
 		metadataClient := http.DefaultClient
-		metadataClient.Timeout = 1 * time.Minute
+		metadataClient.Timeout = 30 * time.Second
```

Comment:
I wonder if you were right the first time. Hard to know what slow network is going to do :| 

And I guess this is non-blocking